tensor_test.go

```diff
@@ -10,3 +10,17 @@ func ExampleTensor() {
 	t.Close()
 	// Output:
 }
+
+func ExampleTranspose2d() {
```

Comment:
Please rename the function name should begin with **Test**  word, please rename to `TestExampleTranspoose2d(t  
 *testing.T)` .